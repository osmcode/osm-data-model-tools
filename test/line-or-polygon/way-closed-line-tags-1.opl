w1 Nn1,n2,n1 Thighway=residential
