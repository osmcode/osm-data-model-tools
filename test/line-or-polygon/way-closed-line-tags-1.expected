Statistics:
  non-closed: 0
  closed:     1 (100%)
    unknown:    0 (0%)
    linestring: 1 (100%)
    polygon:    0 (0%)
    both:       0 (0%)
    no tags:    0 (0%)
    error:      0 (0%)
Found 0 unknown keys.
Unknown keys:
